README.md

```diff
@@ -227,8 +227,8 @@ Default: `process.cwd()`
 Ability to set the current working directory. Defaults to `process.cwd()`. Can either be a string to set the cwd to match files and spawn tasks. Or an object to set each independently. Such as `options: { cwd: { files: 'match/files/from/here', spawn: 'but/spawn/files/from/here' } }`.
 
 #### options.livereloadOnError
-Type: `Boolean`  
-Default: `true`  
+Type: `Boolean`
```

Comment:
edit the docs file that builds this file (true for all below changes). example is edit https://github.com/gruntjs/grunt-contrib-watch/blob/master/docs/watch-examples.md